ci/prow/config.yaml

```diff
@@ -53,11 +53,15 @@ deck:
         name: buildlog
         config:
           highlight_regexes:
-          - 'panic: test timed out after.*$'
           - timed out
```

Comment:
Since you're here, what about removing the loose "timed out" and "panic" entries, and moving "ERROR:" to next to "FAILL:"?

Reply:
As per discussion offline, `timed out` is useful under certain situations so we still keep it, also separate the regexes into two groups to make them more clear.